lib/output.js

```diff
@@ -13,10 +13,15 @@ const formats = new Map([
   ['raw', 'raw'],
   ['tiff', 'tiff'],
   ['webp', 'webp'],
-  ['gif', 'gif']
+  ['gif', 'gif'],
+  ['jp2', 'jp2'],
+  ['jpx', 'jp2'],
+  ['j2k', 'jp2'],
+  ['j2c', 'jp2']
 ]);
 
 const errMagickSave = new Error('GIF output requires libvips with support for ImageMagick');
+const errJp2Save = new Error('JP2 output requires libvips with support for openjpeg');
```

Comment:
Might need to change `openjpeg` to `OpenJPEG` as the unit tests match on `/JP2 output requires libvips with support for OpenJPEG/`